include/oembed.php

```diff
@@ -255,12 +257,14 @@ function oembed_format_object($j){
  * @see oembed_format_object()
  */
 function oembed_iframe($src, $width, $height) {
+	$a = get_app();
+
 	if (!$height || strstr($height,'%')) {
 		$height = '200';
 	}
 	$width = '100%';
 
-	$s = App::get_baseurl() . '/oembed/'.base64url_encode($src);
+	$s = $a->get_baseurl() . '/oembed/'.base64url_encode($src);
```

Comment:
@annando Please take a look at that function (get_baseurl()). It wraps a static call to a object-referencing call.